tests/integration/helpers/cluster.py

```diff
@@ -1448,10 +1466,9 @@ def create_dir(self, destroy_dir=True):
             self._create_odbc_config_file()
             odbc_ini_path = '- ' + self.odbc_ini_path
 
-        entrypoint_cmd = CLICKHOUSE_START_COMMAND
-
```

Comment:
It's better to move this line back here.

Reply:
Ok